app/views/notification_mailer/meeting_reminder.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>
+  Hi <%= @member.name %>,
+</p>
+<p>
+Here's a friendly reminder that your meeting tomorrow, "<%= @meeting.name%>", will be taking place at <%= @meeting.time %> at 
```

Comment:
Let me know what you actually want this to say.  I kinda just made this up and I'm no copywriter.. 😸